pkgs/development/ocaml-modules/jingoo/default.nix

```diff
@@ -1,26 +1,25 @@
-{ stdenv, fetchFromGitHub, ocaml, findlib, pcre, uutf }:
+{ lib, buildDunePackage, fetchFromGitHub
+, menhir, ppx_deriving, re, uutf, uucp, ounit2 }:
 
-if !stdenv.lib.versionAtLeast ocaml.version "4.02"
-then throw "jingoo is not available for OCaml ${ocaml.version}"
-else
+buildDunePackage rec {
+  pname = "jingoo";
+  version = "1.3.4";
 
-stdenv.mkDerivation rec {
-  name = "ocaml${ocaml.version}-jingoo-${version}";
-  version = "1.2.18";
+  minimumOCamlVersion = "4.04";
 
   src = fetchFromGitHub {
     owner = "tategakibunko";
     repo = "jingoo";
     rev = "v${version}";
-    sha256 = "0gciiysrjy5r4yiisc41k4h0p530yawzqnr364xg8fdkk444fgkn";
+    sha256 = "0fsmm6wxa3axwbcgwdidik3drg754wyh2vxri2w12d662221m98s";
   };
 
-  buildInputs = [ ocaml findlib ];
-  propagatedBuildInputs = [ pcre uutf ];
+  propagatedBuildInputs = [ menhir ppx_deriving re uutf uucp ];
```

Comment:
`menhir` does not need to be propagated.

Reply:
fixed